test/unit/events_tests.js

```diff
@@ -0,0 +1,115 @@
+var test = require('../lib/ptaptest')
+var qs = require('querystring')
+var config = require('../../lib/config')
+var TestServer = require('../lib/test_server')
+var r = require('../../lib/prequest')
+
+var keys = require('../../lib/keys')(config)
+var base = 'http://' + config.server.host + ':' + config.server.port
+var publish = base + '/v0/publish'
+var events = base + '/v0/events'
+var authHeaders = {
+  'Authorization': 'Bearer foo-bar-notify'
+}
+
+var fooJWT = keys.secret.signSync(
+  {
+    uid: 'foo',
+    rid: 'bar',
+    typ: 'baz'
+  }
+)
+
+function publishFoo(t) {
+  return r.postAsync(
+    {
+      url: publish,
+      json: {
+        events: [fooJWT]
+      }
+    }
+  )
+  .then(
+    function (result) {
+      t.equal(result[0].statusCode, 200)
+    }
+  )
+}
+
+TestServer.start(config)
+.then(function (server){
+
+  test(
+    'matching all filters',
+    function (t) {
```

Comment:
One thing I like from the auth-db-server, is the use of `t.plan(X)` to declare how many assertions we expect to trigger.  It's caught accidentally-passing tests for me a couple of times.  Thoughts?

Definitely follow-up-bug material though.